src/Microsoft.DotNet.Build.CloudTestTasks/SendJobsToHelix.cs

```diff
@@ -55,8 +54,12 @@ public override bool Execute()
         private async Task<bool> ExecuteAsync()
         {
             List<ITaskItem> jobIds = new List<ITaskItem>();
-            string joinCharacter = ApiEndpoint.Contains("?") ? "&" : "?";
-            string apiUrl = ApiEndpoint + joinCharacter + "access_token=" + Uri.EscapeDataString(AccessToken);
+            string apiUrl = ApiEndpoint;
+            if (!String.IsNullOrEmpty(AccessToken))
+            {
+                string joinCharacter = ApiEndpoint.Contains("?") ? "&" : "?";
```

Comment:
is there ever a case where the API endpoint already has a query string that isn't the access token?